backend/app/api/src/helpers/UitpasTokenRepository.ts

```diff
@@ -0,0 +1,102 @@
+import { SimpleError } from '@simonbackx/simple-errors';
+import { UitpasToken } from '@stamhoofd/models';
+
+export class UitpasTokenRepository {
+    accessToken?: string;
+    expiresOn: Date = new Date(0); // Set to minimum time initially
+    uitpasToken: UitpasToken;
+
+    constructor(uitpasToken: UitpasToken) {
+        this.uitpasToken = uitpasToken;
+    }
+
+    static knownTokens: Map<string | null, UitpasTokenRepository> = new Map(); // organizationId (null means platform) -> UitpasToken
```

Comment:
Als je documentatie schrijft is het vaak handig om 
```
/** 
 *... 
 */
``` 
te gebruiken, en de comment voor je method of property te zetten. Als je dan in je IDE hovert over de property of method, dan zal je deze beschrijving zien.